client/my-sites/upgrades/checkout-thank-you/header.jsx

```diff
@@ -7,11 +7,12 @@ import React from 'react';
 /**
  * Internal dependencies
  */
-import { isFreeTrial, isPlan } from 'lib/products-values';
+import { isDomainRegistration, isFreeTrial, isPlan } from 'lib/products-values';
 import Gridicon from 'components/gridicon';
 
 const CheckoutThankYouHeader = React.createClass( {
 	propTypes: {
+		domain: React.PropTypes.string,
```

Comment:
Again as mentioned in https://github.com/Automattic/wp-calypso/pull/3715#discussion_r54708089 I think we could retrieve the domain name from `primaryPurchase.meta`.